lib/tcti/tpm2_tools_tcti_device.c

```diff
@@ -0,0 +1,44 @@
+#include <inttypes.h>
+#include <stdlib.h>
+#include <tcti/tcti-tabrmd.h>
+
+#include <sapi/tpm20.h>
+#include <tcti/tcti_device.h>
+
+#include "log.h"
+#include "tpm2_util.h"
+
+#define TPM2TOOLS_ENV_DEVICE_FILE    "TPM2TOOLS_DEVICE_FILE"
+#define TCTI_DEVICE_DEFAULT_PATH "/dev/tpm0"
+
+TSS2_TCTI_CONTEXT *tpm2_tools_tcti_device_init(char *opts) {
+    TCTI_DEVICE_CONF conf = {
+        .device_path = opts,
```

Comment:
Shouldn't this be .device_path = TCTI_DEVICE_DEFAULT_PATH ?

Reply:
Yeah none of the tcti initialization is even close to being done correctly.